web-client/public/js/setup-handlers.js

```diff
@@ -110,38 +108,39 @@ export const setupHandlers = grnState => {
                     }
                 }
             };
-            visit(svg);
+            visit(node);
             return tree;
         };
 
         const explicitlySetStyle = element => {
-            const cSSStyleDeclarationComputed = window.getComputedStyle(element);
-            let i;
-            let len;
-            let key;
-            let value;
-            let computedStyleStr = "";
-
-            for (i = 0, len = cSSStyleDeclarationComputed.length; i < len; i++) {
-                key = cSSStyleDeclarationComputed[i];
-                value = cSSStyleDeclarationComputed.getPropertyValue(key);
+            const cssStyleDeclarationComputed = window.getComputedStyle(element);
+            const computedStyleObj = {};    
+
+
+            for (let i = 0; i < cssStyleDeclarationComputed.length; i++) {
+                const key = cssStyleDeclarationComputed[i];
+                const value = cSSStyleDeclarationComputed.getPropertyValue(key);
```

Comment:
Whoops, shouldn’t this be edited also?
```suggestion
                const value = cssStyleDeclarationComputed.getPropertyValue(key);
```

Reply:
I didn't catch this one. Usually, I would use Ctrl + D to select all words, but I think I was fixing it manually. But I would love to learn about VS Code's Rename command!